test/robot/web/client/widgets/grid/data.go

```diff
@@ -114,20 +117,24 @@ func (l TaskList) stats() taskStats {
 		numCurrentSucceeded:       l.Count(taskCurrentSucceeded),
 		numStaleSucceeded:         l.Count(taskStaleSucceeded),
 		numInProgressWasSucceeded: l.Count(taskInProgressWasSucceeded),
+		numFixed:                  l.Count(taskFixed),
```

Comment:
nit: I'd stick with the existing naming for consistency's sake:
`numSucceededWasFailed`, etc.